packages/core/src/baseclient.ts

```diff
@@ -309,6 +340,46 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
 
   /** @inheritdoc */
   public init(): void {
+    if (this.config) {
+      const remoteConfigFetchPromise = this.config.fetch();
+      // TODO calling _setupIntegrations async will break the world
+      // const { blockForRemoteConfig } = this._options;
+      // If `blockForRemoteConfig` is configured and there is no cached config, then
+      // if (this.config.getSource() === 'DEFAULT' && blockForRemoteConfig && blockForRemoteConfig.timeout > 0) {
+      //   new Promise(resolve => {
+      //     setTimeout(resolve, blockForRemoteConfig.timeout);
+      //     remoteConfigFetchPromise.then(resolve);
+      //   })
+      //     .then(() => this.finishInit())
+      //     .catch(() => {
+      //       // TODO: Error handling
+      //     });
+      //
+      //   return;
+      // }
```

Comment:
Can't do this 😬 , we'll have to apply new configs and have sdks restart if possible.